nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -41,6 +41,8 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 - `nitter` requires a `guest_accounts.jsonl` to be provided as a path or loaded into the default location at `/var/lib/nitter/guest_accounts.jsonl`. See [Guest Account Branch Deployment](https://github.com/zedeus/nitter/wiki/Guest-Account-Branch-Deployment) for details.
 
+- `services.aria2.rpcSecret` has been replaced with `services.aria2.rpcSecretFile`.
```

Comment:
Could you add a bit more information to help users migrate? As an example you can check the release note 19.09 for `services.gitlab`

https://github.com/NixOS/nixpkgs/blob/24bd0c9a97a61b7cc9b8676155e0067e119cd00c/nixos/doc/manual/release-notes/rl-1909.section.md?plain=1#L213